ucp/public_api.py

```diff
@@ -177,7 +183,7 @@ def get_config():
     """
 
     if _ctx is None:
-        return core.get_config()
+        return ucx_api.get_default_options()
```

Comment:
Will this do the right thing? What if environment variables are passed, will those be read?

Reply:
Yes, it calls `ucp_config_print()`, which also reads environment variables but I can see that the name is very misleading :)
Renamed it to `get_current_options()`

Reply:
Ok, this is what I thought, indeed renaming makes more sense. Thanks for adjusting that!